.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
         matrix:
-          node-version: ['lts/fermium', 'lts/gallium']
```

Comment:
This step should use all current LTS versions

Reply:
So that will run `ci` on each of those node versions to ensure that we are backwards compatible should someone be running this locally on one of those versions right?

Reply:
https://github.com/nodejs/release#release-schedule

fermium is already end of life, so remove it. I thought gallium was end of life in October, but it seems like they moved it up to Sept 11. Given that being just a few weeks away, lets remove that too.

But 20 should be added, I'm not sure if `lts/iron` will work or not, but try using the codename and seeing if it works before reverting to 20 (https://github.com/nodejs/Release/blob/main/CODENAMES.md)

Reply:
Looks like lts/iron was not recognized. Tried the ci build on my [fork](https://github.com/eudoroolivares2016/edsc-geo/actions/runs/5928274529/job/16073528160) so using node-20